spec/components/app_vaccination_record_api_sync_status_component_spec.rb

```diff
@@ -61,6 +61,22 @@
           )
         end
       end
+
+      context "when child requested that parents aren't notified" do
+        before do
+          allow(vaccination_record).to receive_messages(
+            sync_status: :not_synced,
+            notify_parents: false
+          )
```

Comment:
```suggestion
          vaccination_record.sync_status = "not_synced"
          vaccination_record.notify_parents = false
```

Does this work as an alternative?

Reply:
Most likely. What's the benefit? It seems like exactly the same stubbing logic, but just not using the stubbing syntax